AdobeExtensionSample/readme.md

```diff
@@ -0,0 +1,23 @@
+
+# ForeSee Adobe Extension Sample
+
+## Prerequisites
+
+To run this Sample, you need a Adobe Experience Platform Launch account, and have set it up to use in mobile Apps.  
+
+## Installing
+1.  In the Adobe Launch dashboard, create a new Property for Mobile platform.
```

Comment:
Should we link to the platform here? It's probably obvious, but it might make it a bit easier on clients.

Reply:
When you create a property, you can specify whether it's for the Web platform or for the Mobile platform. I don't think it's possible to insert a link there to the platforms though

Reply:
@awangvan can you add the link to the Adobe Launch homepage, as we discussed?

Reply:
Done